src/ansys/mapdl/core/_commands/session/run_controls.py

```diff
@@ -20,670 +20,1147 @@
+        files including ``\*DO``, :ref:`use`, :ref:`ulib`, and the "Unknown Command" Macro have less nesting
+        available because each of these operations also uses a level of file switching. For an interactive
+        run, a :ref:`input`,TERM switches to the terminal for the next input. A ``/EOF`` read from the
+        terminal then switches back to the previous file. A :ref:`input` (with a blank second field)
+        switches back to the primary input file.
+
+        Setting ``LOG`` = 1 on :ref:`input` causes all commands read from the specified file to be recorded
+        in the command log ( :file:`File.LOG` ) and the internal database command log ( :ref:`lgwrite` ).
+        This option is recommended if the log file will be used later. The ``LOG`` = 1 option is only valid
+        when the :ref:`input` occurs in the primary input file. Using ``LOG`` = 1 on a nested :ref:`input`
+        or on a :ref:`input` within a do-loop will have no effect (that is, commands in the secondary input
+        file are not written to the command log).
+
+        The ``Dir`` option is optional as the directory path can be included directly in ``Fname``.
 
         This command is valid in any processor.
         """
-        return self.run(f"/STATUS,{lab}", **kwargs)
+        command = f"/INPUT,{fname},{ext},{dir_},{line},{log}"
+        return self.run(command, **kwargs)
 
-    def starstatus(
-        self,
-        par="",
-        imin="",
-        imax="",
-        jmin="",
-        jmax="",
-        kmin="",
-        kmax="",
-        lmin="",
-        lmax="",
-        mmin="",
-        mmax="",
-        kpri="",
-        **kwargs,
-    ):
-        """Lists the current parameters and abbreviations.
+    def cwd(self, dirpath: str = "", **kwargs):
+        r"""Changes the current working directory.
+
+        Mechanical APDL Command: `/CWD <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en//ans_cmd/Hlp_C_CWD_sl.html>`_
+
+        Parameters
+        ----------
+        dirpath : str
+            The full path name of the new working directory.
+
+        Notes
+        -----
+
+        .. _s-CWD_notes:
+
+        After issuing the /CWD command, all new files opened with no default directory specified (via the
+        :ref:`file`, :ref:`copy`, or :ref:`resume` commands, for example) default to the new ``DIRPATH``
+        directory.
+
+        Examples
+        --------
+        Change MAPDL's working directory to ``"C:/temp"``.  This
+        assumes that MAPDL running on Windows.
+
+        >>> mapdl.cwd("C:/temp")
+
+        MAPDL on Linux example:
+
+        >>> mapdl.cwd("/tmp/")
+        """
+        dirpath = str(dirpath)
+        if not (dirpath.startswith("'") and dirpath.endswith("'")) and "'" in dirpath:
+            raise MapdlRuntimeError(
+                'The CWD command does not accept paths that contain singular quotes "'
```

Comment:
```suggestion
                'The CWD command does not accept paths that contain singular quotes "\'"'
```

Reply:
This comes from the pymapdl ``main`` branch.

Reply:
https://github.com/ansys/pymapdl/pull/3938 should fix it!